torch/optim/_multi_tensor/rmsprop.py

```diff
@@ -28,10 +28,12 @@ class RMSprop(Optimizer):
         centered (bool, optional) : if ``True``, compute the centered RMSProp,
             the gradient is normalized by an estimation of its variance
         weight_decay (float, optional): weight decay (L2 penalty) (default: 0)
+        lr_in_momentum (bool, optional): learning rate scaling is included in the
+            momentum buffer update
 
     """
 
-    def __init__(self, params, lr=1e-2, alpha=0.99, eps=1e-8, weight_decay=0, momentum=0, centered=False):
+    def __init__(self, params, lr=1e-2, alpha=0.99, eps=1e-8, weight_decay=0, momentum=0, centered=False, lr_in_momentum=False):
```

Comment:
This is not standard in pytorch, and doesn't seem like a standard type of argument for optimizers. What are alternatives?

Is the goal only to improve performance? In that case it would be good to have a benchmark.

Reply:
Thanks, make sense. I will post the perf result.

Reply:
@vincentqb  could you help to review the extension and also the numeric change?

Reply:
For the API change, I do not see a reason to add this in the API of RMSProp. No other optimizers have that.

If there is a runtime speed improvement for running the step, I don't believe this would justify adding this API, since this is still a change in the RMSProp algorithm we have.

As I mention in [comment](https://github.com/pytorch/pytorch/pull/46118#issuecomment-718264930), if there is a discrepancy between RMSProp with and without multitensor, this needs to be investigated.

Reply:
The change relates to #23796, and the difference between pytorch and tensorflow -- and not between the original implementation and the multi-tensor.